themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```
@@ -399,6 +401,12 @@ logout-service-post-binding-url=Logout Service POST Binding URL
 logout-service-post-binding-url.tooltip=SAML POST Binding URL for the client's single logout service. You can leave this blank if you are using a different binding
 logout-service-redir-binding-url=Logout Service Redirect Binding URL
 logout-service-redir-binding-url.tooltip=SAML Redirect Binding URL for the client's single logout service. You can leave this blank if you are using a different binding.
+logout-service-artifact-binding-url=Logout Service Artifact Binding URL
+logout-service-artifact-binding-url.tooltip=SAML Artifact Binding URL for the client's single logout service. You can leave this blank if you are using a different binding.
+artifact-binding-url= Artifact Binding URL
+artifact-binding-url.tooltip= SAML Artifact Binding URL for the client's artifact binding service. You can leave this blank if the standard return address the same as the artifact binding service. This value should be set when forcing artifact binding together with IdP initiated login.
```

Could this be rephrased? I got lost when reading the hint.
```suggestion
artifact-binding-url.tooltip=URL to send the HTTP ARTIFACT messages to. You can leave this blank if you are using a different binding. This value should be set when forcing ARTIFACT binding together with IdP initiated login.
```